src/components/GenreTab/GenreTab.js

```diff
@@ -0,0 +1,17 @@
+import React, { useContext } from 'react';
+import styles from './GenreTab.css';
+import { MainContext } from '../Main/Main';
+
+const GenreTab = ({ isActive, children }) => {
+  const { filterByGenreHandle } = useContext(MainContext);
+  return (
+    <button
+      onClick={() => filterByGenreHandle(children)}
+      className={`${styles.button} ${isActive ? styles.active : ''}`}
```

Comment:
Useful package for className https://www.npmjs.com/package/classnames

Reply:
thanks!